velox/substrait/proto/substrait/algebra.proto

```diff
@@ -194,6 +194,10 @@ message FetchRel {
   int64 offset = 3;
   // the amount of records to return
   int64 count = 4;
+  // A Boolean indicating whether the node generates partial
+  // results on local workers or finalizes the partial results.
+  bool is_partial = 5;
```

Comment:
Is this a copy of the latest version from Substrait or a manual change? If latter, let's instead update to latest version (in a separate PR before this one).

Reply:
Remove the change of this since we couldn't push pr to substrait repo cause some internal reasons. Add TODO in the code now and will update it after the substrait merged the related field.